javascript/javascript2/week2/movies2.js

```diff
@@ -0,0 +1,85 @@
+console.log("script loaded");
+console.log(movies);
+
+//with short names
+const shortName = movies.filter(movie => movie.title.length < 5)
+.forEach(movie => console.log(movie.title));
+
+//with long names
+const longName = movies.filter(movie => movie.title.length > 15)
+.forEach(movie => console.log(movie.title));
+
+//80s movies
+const eighties = movies.filter(movie => movie.year >= 1980 && movie.year <= 1989)
+.forEach(movie => console.log(movie.title));
+
+//Movies with tags
+const moviesWithTag = movies.map(movie => {
+    if (movie.rating >= 7){
+        tag = 'Good';
+    } else if (movie.rating >= 4){
+        tag = 'Average';
+    } else {
+        tag = 'Bad';
+    }
+    return {...movie, tag: tag};                        
+  });
+
+console.log(moviesWithTag);
+
+//Chaining
+const highRatings = movies.filter(movie => movie.rating > 6)
+.map(movie => movie.rating);
+console.log(highRatings);
+
+//Find keyWords
+const keywords = ['Surfer', 'Alien', 'Benjamin'];
+keywords.forEach(keyword => {
+  let keywordsEach = keyword.toLowerCase();
+  const moviesWithKeyWords = movies.filter(movie => movie.title
+    .toLowerCase()
+    .includes(keywordsEach));
+    console.log(moviesWithKeyWords.length)
+  })
```

Comment:
Did you check `console.log` on line no 42?

Question is Count the total number of movies containing any of following keywords: Surfer, Alien or Benjamin. So if there were 3 movies that contained Surfer, 1 with Alien and 2 with Benjamin, you would return 6. Can you make sure the search is case insensitive?

I think your solution will console log count for each keyword.  Question specified total count for all  given keywords. You almost solved the question.  Try to get total count .